noxfile.py

```diff
@@ -27,8 +27,8 @@
 BLACK_PATHS = ["docs", "google", "tests", "noxfile.py", "setup.py"]
 
 DEFAULT_PYTHON_VERSION = "3.8"
-SYSTEM_TEST_PYTHON_VERSIONS = ["2.7", "3.8"]
-UNIT_TEST_PYTHON_VERSIONS = ["2.7", "3.5", "3.6", "3.7", "3.8"]
+SYSTEM_TEST_PYTHON_VERSIONS = ["3.8"]
+UNIT_TEST_PYTHON_VERSIONS = ["3.6", "3.7", "3.8", "3.9"]
```

Comment:
Your comment says that you weren't including these changes, which should be driven by synthtool.

Reply:
#313 indeed does update `noxfile.py` via synth.

Reply:
Fixed when resolving conflict with #313.